urlscan/help.md

```diff
@@ -34,12 +34,17 @@ This action is used to search urlscan.io.
 
 ##### Input
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|offset|integer|0|True|Offset of first result (for paginating)|None|
-|q|string|None|True|The query term (ElasticSearch simple query string) e.g domain:urlscan.io, default is *|None|
-|size|integer|100|True|Number of results returned|None|
-|sort|string|_score|True|Sorting, specificied via $sort_field:$sort_order|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|offset|integer|0|True|Offset of first result (for paginating)|None|None|
+|q|string|None|True|The query term (ElasticSearch simple query string) e.g domain:urlscan.io, default is *|None|None|
+|size|integer|100|True|Number of results returned|None|None|
+|sort|string|_score|True|Sorting, specificied via $sort_field:$sort_order|None|None|
+
+Example input:
+
+```
```

Comment:
Missing raw example input

Reply:
> This PR is missing just about all the example inputs sections and keys in the spec. It also needs a major version bump

I am not happy to confess this, but the reason for this is because a second help.md file got in.  Removed it from the branch and rebased but that clears up all these.